graphdatascience/resources/imdb/serialize_imdb.py

```diff
@@ -0,0 +1,68 @@
+from importlib.resources import path
+
+import pandas as pd
+from pandas import read_pickle
+
+"""
+This code included only for reproducibility.
+
+It is used to convert raw pickle data from edges.pkl, labels.pkl and node_features.pkl
+into consolidated imdb_nodes_gzip.pkl and imdb_rels_gzip.pkl.
+
+These pickles are then being read by load_imdb()
+"""
+
+with path("graphdatascience.resources.imdb", "raw/labels.pkl") as labels_resource:
+    class_labels = read_pickle(labels_resource)
+movies = pd.DataFrame([item for sublist in class_labels for item in sublist])
+movies = movies.rename(columns={0: "nodeId", 1: "class"})
+movies["labels"] = "Movie"
+
+with path("graphdatascience.resources.imdb", "raw/node_features.pkl") as features_resource:
+    raw_features = read_pickle(features_resource)
+nodes_features_df = pd.DataFrame(raw_features)
+nodes_features_df["feature"] = nodes_features_df.values.tolist()
+nodes_features_df = nodes_features_df["feature"].reset_index().rename(columns={"index": "nodeId"})
+
+movie_nodes_with_features = movies.merge(nodes_features_df, on="nodeId", how="left")
+person_nodes_with_features = (
+    pd.merge(nodes_features_df, movies, on=["nodeId"], how="left", indicator=True)
+    .query('_merge=="left_only"')
+    .drop(columns=["class", "_merge"])
+)
+person_nodes_with_features["labels"] = "Person"
+# Set 'Person' class as -1.0 since gds.graph.construct only allows one nodeDF for community,
+# and that setting NaN gives value is null error from Arrow flight RPC
```

Comment:
Talking with Martin, `NaN` we expect to support.
Setting values to `null` we cannot, as what is the primitive version of `null` after all

Reply:
Tried `np.nan` and `float("NaN")`, neither seems to work... Any other way of doing it?

Reply:
I think it could be nice to somehow separate movies that have class labels, and those that have not. For example, in addition to the "Movie" label, they could have either "Labeled" or "Unlabeled" labels. This way we would be able to use the dataset in our NC pipeline (which one of the reasons we are doing this), and you don't really have to worry about null/nan features. Quite simply, only ["Movie", "Labeled"] nodes would have the class label.
This is similar to what we're doing with Cypher in our quality benchmarks on this dataset.

Reply:
can you card this @brs96 , that np.nan() does not work with arrow? I think its worth to investigate more (but not on this PR as I agree with Adam)

Reply:
I think we still need NaN support for this since since Cypher loading does not support using several data frames. So they must all have the label property

Reply:
NaN support was expected to work and we will look into why the python NaN does not work